compiler/sem/sem.nim

```diff
@@ -974,6 +974,14 @@ proc appendInstancedGenericRuntimeRoutines(c: PContext, n: PNode) =
         unreachable()
   c.lastGenericIdx = c.generics.len
 
+proc sealRodFile(c: PContext) =
```

Comment:
Definitely better, moving away from wide (interface) and/or shallow (minimal implementation/decision encapsulation) is the way to go.